docs: Nextflow version selector (#1651)
seqeralabs/docs: platform-cloud/docs/launch/advanced.md

- Use the launch form's **Nextflow config file** field for settings that must take precedence over everything else.
:::

## Nextflow version

> Select the Nextflow version used to execute the run. The selector lists the versions available in your installation and resolves the choice to the launch container image that runs the workflow.

Reviewer: ```suggestion

Select the Nextflow version for the run. The selector lists the versions available in your installation and maps your choice to the launch container image that runs the workflow.
```

The default version is:

- **Pipeline advanced options**: the system default version, or the compute environment type's minimum version when that minimum is higher.
- **Launch advanced options**: the version saved on the pipeline, when it is compatible with the selected compute environment. If the pipeline's saved version is below the minimum required by the compute environment, no version is preselected and you must choose a compatible version before the launch can proceed.

Version availability depends on the compute environment:

> - **Cloud and Kubernetes** compute environments (AWS Batch, Azure Batch, Google Batch, Kubernetes) support version selection. Versions below the minimum required by the compute environment are not selectable. A launch submitted with a lower or unknown version through any channel (UI, API, or CLI) is rejected before execution.

Reviewer: ```suggestion
- **Cloud and Kubernetes** compute environments (AWS Batch, Azure Batch, Google Batch, Kubernetes) support version selection. You cannot select versions below the compute environment's minimum. A launch submitted with a lower or unknown version through any channel (UI, API, or CLI) is rejected before execution.
```

> - **Grid/HPC** compute environments (Slurm, LSF, Grid Engine, Altair PBS Pro, Moab) run a pre-installed Nextflow and have no launch container, so the version selector is not shown. A version carried over from a pipeline default has no effect when the pipeline is launched on a grid environment.

Reviewer: ```suggestion
- **Grid/HPC** compute environments (Slurm, LSF, Grid Engine, Altair PBS Pro, Moab) run a pre-installed Nextflow and have no launch container. The version selector does not appear for them, and a version carried over from a pipeline default has no effect when you launch on a grid environment.
```
